### PR TITLE
Fix seeds by adding service to the Ad generator

### PR DESCRIPTION
**Pull request by mmorava**
Service field has been added to the Ad model with validations for it to be present within a range of specific values.

This PR updates the seed file so that it generates service field when generating ads.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -18,7 +18,8 @@
     email: rand(0..number_of_ads) < 40 ? FFaker::Internet.email : nil, # 40% chance ad has email
     kind: Ad::KINDS.sample,
     consent: true,
-    address: FFaker::Address.street_name
+    address: FFaker::Address.street_name,
+    service: Ad::SERVICES[rand(Ad::SERVICES.count-1)]
```
Comment (vfonic) [inline db/seeds.rb:22]:
```suggestion
    service: Ad::SERVICES.sample
```

**Review by vfonic — APPROVED** [review]
LGTM! :shipit:

**Review by berislavbabic — APPROVED** [review]
Works perfectly, great job!

**Comment by Macxim** [comment]
Thanks a lot!

**Pull request merged**